### PR TITLE
mention preprint repo in FAQ. Fixes #3981

### LINKED ISSUE #3981: Open access for numba ACM proceedings?
Currently http://numba.pydata.org/numba-doc/latest/user/faq.html?highlight=acm#how-do-i-reference-cite-acknowledge-numba-in-other-work links to http://dl.acm.org/citation.cfm?id=2833162&dl=ACM&coll=DL 

That's behind a paywall - many researchers will not be able to access it, or not want to make an account there and pay for it. So this is non-ideal.

Could you please check if there is a way for you to post a copy of the PDF publicly, e.g. on http://numba.pydata.org or on a preprint server?

If it's forbidden by ACM, you could consider writing something new in the future in an open-access journal?

### PR DESCRIPTION
**Pull request by esc**

### CODE REVIEW

File: docs/source/user/faq.rst [hunk 1]
```diff
@@ -276,4 +276,6 @@ How do I reference/cite/acknowledge Numba in other work?
 --------------------------------------------------------
 For academic use, the best option is to cite our ACM Proceedings:
 `Numba: a LLVM-based Python JIT compiler.
-<http://dl.acm.org/citation.cfm?id=2833162&dl=ACM&coll=DL>`_
+<http://dl.acm.org/citation.cfm?id=2833162&dl=ACM&coll=DL>`_ You can also find
+`a preprint on github <https://github.com/numba/Numba-SC15-Paper>`_ in case you
```
Comment (stuartarchibald) [inline docs/source/user/faq.rst]: Perhaps link to the actual PDF?

Comment (esc) [reply docs/source/user/faq.rst]: Done.

Comment (stuartarchibald) [reply docs/source/user/faq.rst]: Great thanks.

**Comment by stuartarchibald** [comment]
Thanks for the patch, this can be marked for merge once CI passes.

**Pull request merged**